### PR TITLE
fix(package): auditDep now correct calls sub folders

### PR DESCRIPTION
**Pull request by nauynix**
## Problem and Solution

`npm run auditDep` was throwing an error. Upon further inspection it seems that the command was ill-formed.

### CODE REVIEW

**Comment by nauynix** [comment]
In fact... it was complaining that the command was _not found_ 😁

**Comment by LinHuiqing** [comment]
Aye nice, I was just thinking that the 404 was a missed opportunity haha

File: package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "client": "npm start --prefix client",
     "dev": "concurrently \"./wait-for-it.sh localhost:3306 -t 0 -- npm run build-shared && sleep 5 && npm run server\" \"npm run client\" \"docker-compose up \"",
     "postinstall": "npm --prefix server install && npm --prefix client install && npm --prefix shared install && npm run build-shared",
-    "auditDep": "npm audit fix && npm --prefix server audit fix && npm --client server audit fix && npm --shared audit fix",
+    "auditDep": "npm audit fix && npm --prefix server audit fix && npm --prefix client audit fix && npm --prefix shared audit fix",
```
Comment (LoneRifle) [inline package.json]:
```suggestion
    "audit-dep": "npm audit fix && npm --prefix server audit fix && npm --prefix client audit fix && npm --prefix shared audit fix",
```

**Review by LoneRifle — APPROVED** [review]
lgtm otherwise

**Pull request merged**